tests/integration_tests/functional/test_balloon.py

```diff
@@ -700,3 +706,90 @@ def _test_snapshot_compatibility(context):
     )
 
     microvm.kill()
+
+
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Not supported yet."
+)
+def test_memory_scrub(
+    network_config,
+    bin_cloner_path
+):
+    """Test that the memory is zeroed after deflate."""
+    logger = logging.getLogger("snapshot_sequence")
```

Comment:
Not really understanding the need to use snapshotting for making sure memory is zeroed out after deflate?

Reply:
It was the wrong logger call. Now it uses `logging.getLogger()`, which returns the root logger.